examples/api/test/widgets/layout_builder/layout_builder.0_test.dart

```diff
@@ -0,0 +1,19 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter_api_samples/widgets/layout_builder/layout_builder.0.dart' as example;
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  testWidgets('layout_builder.0 Golden', (WidgetTester tester) async {
```

Comment:
Creating the first test as a golden image test is likely to ensure that all samples end up with such a test and the disadvantages of golden image testing are well known.  Maybe we want to make the example test archetype check specific aspects of the LayoutBuilder example?